### DISCUSSION TITLE
Is the team maintainer role scoped to that team

### DISCUSSION THREAD

**Discussion by acgetchell** [community/community#113198] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Is the team maintainer role scoped to that team, or does it apply to the entire organization?

**Comment by vineeth-au** [comment]
Hi acgetchell 👋 ,

> Organization owners can assign roles to individuals and teams giving them different sets of permissions in the organization.

I believe they are not. Perhaps [this](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization) page might have more information on what you are looking for..

**Comment by acgetchell** [comment]
Thanks, I've looked at that page but it does not mention the team maintainer role.

However, a page linked to that one seems to imply that "maintainer" is a repository role.

https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization

**Comment by stjepanDeveloper** [comment, marked as answer]
Hello @acgetchell ,
The Team Maintainer role in GitHub is specifically scoped to the team level, not the entire organization. 
What this means is that a Team Maintainer has certain permissions related to managing team members and team settings, but these permissions do not extend to the entire organization's settings or other teams within the organization. 

Some of the Team maintaier roles :

- adding or removing team members
- creating child teams
- managing team repositories and permissions on those repo's
- editing the team's name, description and privacy settings

However, you don't have access to:

- modifying organization-level settings
- managing all organization repositories
- altering other teams that you're not a maintainer of

This role separation helps in delegating responsibilities without giving away too much control that could potentially affect the whole organization's structure or security.
Hope this helps,

**Reply by acgetchell** [reply]
Awesome, thanks!
